Jenkinsfile

```diff
@@ -89,9 +93,18 @@ node('linux && docker') {
 
       stage('Deployment pipeline upload') {
         lerna = readJSON file: 'lerna.json'
-        prereleaseVersion = lerna.version
-        version = prereleaseVersion.split('-alpha')[0]
-        sh "deployment-package package create --with-deploy --resolve COMMIT_HASH=${commitHash} --resolve VERSION=${version} --resolve PRERELEASE=${prereleaseVersion}  || true"
+        version = lerna.version
+        releaseVersion = version.split('-alpha')[0]
+        sh """\
+          deployment-package package create
+            --with-deploy
+            --package-name ui-kit/v${version}
+            --target-environment ${isRelease ? 'qa' : 'dev'}
+            --resolve COMMIT_HASH=${commitHash}
+            --resolve VERSION=${version}
+            --resolve RELEASE_VERSION=${releaseVersion}
+          || true
+        """.replace('\n', ' ').replaceAll(/\s\s+/, ' ')
```

Comment:
It's not necessary to split everything across multiple lines, but I found it easier to read and review.